examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,796 @@
+        This example also illustrates how to use the
+        <a href="../../#tabpanel">tabs pattern</a>
+        to provide users with a way to skip slides in the sequence by directly choosing which one to view.
+         Navigation among slides is provided by a series of circular buttons that function as tabs and are rendered over the rotating slides.
+        Automatic rotation stops when users move focus to any control or link in the carousel or hover the mouse over carousel content.
+        The accessibility features section that follows the example describes these aspects of the design in detail.
+      </p>
+      <p>Similar examples include:</p>
+      <ul>
+        <li><a href="carousel-1-prev-next.html">Auto-Rotating Image Carousel with Previous and Next Buttons
+        Example:</a> A basic image carousel that demonstrates the accessibility features necessary for carousels that rotate automatically on page load.</li>
+      </ul>
+
+      <section>
+        <h2>Example View and Rotation Options</h2>
+
+        <fieldset class="carousel-options">
+          <legend>Carousel Behavior Options</legend>
+
+          <label for="accessible-controls">
+            <input type="checkbox" id="accessible-controls" value="moreaccessible" aria-describedby="accessible-controls-desc">
+            Tablist carousel controls and captions displayed before and/or after the image.
+          </label>
+          <p id="accessible-controls-desc">This option is more accessible than rendering controls and captions on top of the image, because the controls are harder to perceive and captions are more difficult to read with rotating images behind them.</p>
+
+          <label for="rotation-paused">
+            <input type="checkbox" id="rotation-paused" value="paused" aria-describedby="rotation-paused-desc">
+            Auto-rotation is initially paused (<a href="javascript:history.go(0)"><img class="reload" src="images/reload-icon.png" alt="">reload needed</a>).
+          </label>
+          <p id="rotation-paused-desc">This option controls whether the carousel is paused or playing on page load. The paused option improves accessibility for users with visual impairments and people who are distracted or confused by auto-rotation, but allows users to start auto-rotation using the start/stop button.  If this option is changed, the page must be reloaded to see the effect on behavior.</p>
+
+          <label for="rotation-disabled">
+            <input type="checkbox" id="rotation-disabled" value="norotate" aria-describedby="rotation-disabled-desc">
+            Auto-rotation is disabled.
+          </label>
+          <p id="rotation-disabled-desc">This option improves accessibility for users with visual impairments and people who are distracted or confused by auto-rotation by disabling the auto-rotation feature and removing the start/stop button from the user interface.  Users can use the tablist to manually navigate through the slides.</p>
```

Comment:
Some new words for the checkboxes. Followed [N&N Guideline 7](https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/): 
> Use positive and active wording for checkbox labels

```suggestion
          <label for="accessible-controls">
            <input type="checkbox" id="accessible-controls" value="moreaccessible" aria-describedby="accessible-controls-desc">
            Display controls and captions outside of image
          </label>
          <p id="accessible-controls-desc">This option is more accessible than rendering controls and captions within the image, because controls are easier to perceive and captions are easier to read without rotating images behind them.</p>

          <label for="rotation-paused">
            <input type="checkbox" id="rotation-paused" value="paused" aria-describedby="rotation-paused-desc">
            Pause auto-rotation on load (<a href="javascript:history.go(0)"><img class="reload" src="images/reload-icon.png" alt="">reload needed</a>).
          </label>
          <p id="rotation-paused-desc">This option controls whether the carousel is paused or playing on page load. The paused option improves accessibility for users with visual impairments and people who are distracted or confused by auto-rotation, but allows users to start auto-rotation using the start/stop button.</p>

          <label for="rotation-disabled">
            <input type="checkbox" id="rotation-disabled" value="norotate" aria-describedby="rotation-disabled-desc">
            Disable auto-rotation
          </label>
          <p id="rotation-disabled-desc">This option improves accessibility for users with visual impairments and people who are distracted or confused by auto-rotation by disabling the auto-rotation feature and removing the start/stop button from the user interface. Users can use the tablist to manually navigate through the slides.</p>
```

Reply:
I also think the description text should be indented to align vertically with the label text.

Also, other settings pages that use checkboxes with descriptions usually make the description text lighter than the label text, so we could make the label text bold, so that it is more clearly the "call to action".